src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -698,6 +701,8 @@
           text: az aks create -g MyResourceGroup -n MyManagedCluster --enable-azure-service-mesh
         - name: Create a kubernetes cluster with Azure Monitor Metrics enabled.
           text: az aks create -g MyResourceGroup -n MyManagedCluster --enable-azuremonitormetrics
+        - name: Create a kubernetes cluster with Azure Monitor App Monitoring enabled
+          text: az aks create -g MyResourceGroup -n MyManagedCluster --enable-azure-monitor-app-monitoring
```

Comment:
@jatakiajanvi12 , are we not exposing the open telemetry flags and ports to configure?

Reply:
Will be doing that in a separate PR